guide/app_accuracy.tex

```diff
@@ -253,7 +253,7 @@ \section{Comparison to Reference Data}
 provided by JPL's solutions DE440/DE441 in J2000.0 equatorial
 coordinates, including JPL's corrections for nutation, light time,
 aberration, light deflection in the Solar gravity field, and other
-relativistic effects. The right part provides the according values
```

Comment:
If you think this is in any way better, we can say "respective values". I want to express "these are the results of Stellarium 0.21.2&beta; put next to/adjacent/side-to-side  the results of JPL Horizons for the same input dates."